api/controllers/pr/index.js

```diff
@@ -23,32 +23,57 @@ exports.index = (req, res) => {
   Promise.all([
     findGithubPrs(github, username),
     findGitlabPrs(gitlab, username),
-    github.users.getByUsername({ username }).then(logCallsRemaining),
+    searchGithubUser(github, username),
+    searchGitlabUser(gitlab, username),
   ])
-    .then(([prs, mrs, user]) => {
-      if (user.data.type !== 'User') {
+    .then(([prs, mrs, user, gitlab_user]) => {
+      let isGitHubUser = true;
+      if (user == null) isGitHubUser = false;
+      let isGitLabUser = true;
+      if (JSON.stringify(gitlab_user) == JSON.stringify([]))
+        isGitLabUser = false;
+
+      if (
+        (!isGitHubUser || (isGitHubUser && user.data.type !== 'User')) &&
+        !isGitLabUser
+      ) {
```

Comment:
I think we can prob get rid of the `user.data.type` check?? Or if we do keep it, do we need a similar one for gitlab?

Reply:
we can keep this check assuming github does not change the api. 
This check does not affect gitlab. for gitlab there is separate check !isGitLabUser.

Reply:
Yeah, I suppose type _could_ be `Organization` so maybe worth keeping?

Reply:
we can remove it since you are planning to add some team/organization features. We dont want to indirectly mess with the logic.